include/kq.h

```diff
@@ -173,10 +172,10 @@ class KGame
      * This loads a new map and performs all of the functions that accompany the loading of a new map.
      *
      * \param   map_name Base name of map (xxx -> maps/xxx.map)
-     * \param   player_x New x-coord for player. Pass 0 for msx and msy to use the 'default' position stored in the map file: s_map::stx and s_map::sty
-     * \param   player_y New y-coord for player
-     * \param   camera_x New x-coord for camera. Pass 0 for mvx and mvy to use the default: s_map::stx and s_map::sty)
-     * \param   camera_y New y-coord for camera
+     * \param   player_x New x-coord for player. Pass 0 for msx and msy to use the 'default' position stored in the map
+     * file: s_map::stx and s_map::sty \param   player_y New y-coord for player \param   camera_x New x-coord for
+     * camera. Pass 0 for mvx and mvy to use the default: s_map::stx and s_map::sty) \param   camera_y New y-coord for
+     * camera
```

Comment:
For whatever reason, `clang-format` mangled this but doesn't appear to mangle other `\param ...` areas.

Reply:
It's weird! Option CommentPragmas in https://clang.llvm.org/docs/ClangFormatStyleOptions.html# should be able to fix it.